java/Dockerfile

```
@@ -11,6 +11,8 @@ FROM tomcat:jre17
 
 # Copy WAR file
 COPY --from=builder /build/app/build/libs/vss-1.0.war /usr/local/tomcat/webapps/vss.war
+# Add datadog tracing library for APM
+ADD 'https://dtdg.co/latest-java-tracer' dd-java-agent.jar
```

```suggestion
ADD 'https://dtdg.co/latest-java-tracer' /dd-java-agent.jar
```

Is the root a good place for this?